Sources/RunnerLib/GetDangerJSPath.swift

```diff
@@ -1,13 +1,13 @@
+import DangerShellExecutor
 import Foundation
 import Logger
-import ShellOut
 
-public func getDangerCommandPath(logger: Logger, args: [String] = CommandLine.arguments, shellOutExecutor: ShellOutExecuting = ShellOutExecutor()) throws -> String {
+public func getDangerCommandPath(logger: Logger, args: [String] = CommandLine.arguments, shellOutExecutor: ShellExecuting = ShellExecutor()) throws -> String {
```

Comment:
<!--
  0 failure: 
  1 warning:  Line should be 12...
  
  
  DangerID: danger-id-Danger;
  File: Sources/RunnerLib/GetDangerJSPath.swift;
  Line: 5;
-->

- :warning: Line should be 120 characters or less: currently 159 characters (`line_length`)